extrafiles/tooling/nktooling/__init__.py

```diff
@@ -117,22 +149,53 @@ def isIdentical(self):
 		"""
 		return "".join( self.in_lines ) == "".join( self.out_lines )
 
-	def commit(self, force = False):
+	def commit(self, force = False, outputDiff = None):
 		"""
 		Writes buffered lines into the specified in the constructor.
 		Writes are carried out atomically, i.e. we first write into a temporary file
 		and then replace the original.
-		If the module is currently readonly (which is the default, see setup) no change
-		is carried out.
+		If the module is currently readonly (which is the default, see setup),
+		the original file is not replaced.
 		This function may only be called if not reads are taking place currently (e.g.,
 		via lines())
+
+		If outputDiff is None, its value is copied from doShowDiff(). If it is True,
```

Comment:
A (small) typo in the code-documentation: It should be `doReportDiff()` instead of `doShowDiff()`.